core/src/main/java/org/pentaho/di/core/database/Database.java

```diff
@@ -129,6 +130,8 @@
 
   private DatabaseMeta databaseMeta;
 
+  private static final String DATA_SERVICES_PLUGIN_ID = "KettleThin";
```

Comment:
Not necessary to have this static. It's only used internally in the class. private final should suffice